### PR TITLE
add split wireshark dissectors for avatar/entity protocols

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

File: tools/dissectors/hf-avatar.lua [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+}
+
+local packet_type_extractor = Field.new('hfudt.type')
+
+function p_hf_avatar.dissector(buf, pinfo, tree)
+  pinfo.cols.protocol = p_hf_avatar.name
+
+  avatar_subtree = tree:add(p_hf_avatar, buf())
+
+  i = 0
```
Comment (hyperlogic) [inline tools/dissectors/hf-avatar.lua]: I would prefix this with the local keyword.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10175/HighFidelity-Beta-PR13248-89ea30006dd40ac15a6508fa90df0b7e29983c8f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10175/HighFidelity-Beta-Interface-PR13248-89ea30006dd40ac15a6508fa90df0b7e29983c8f.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10178/HighFidelity-Beta-PR13248-f7710d126fb4165e4b96a0af82bf94be87ba9099.apk

**Comment by hyperlogic** [comment]
No QA is necessary.

**Pull request merged**